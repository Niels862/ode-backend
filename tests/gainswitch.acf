chip {
    io: [
        input,
    ],
    cabs: [
        cab 1 with clocks 1, - {
            cams: [
                GainSwitch as gs1 {},
            ],
        },
    ],
    routing: [
        io1 -> gs1:cmp,
    ]
}
