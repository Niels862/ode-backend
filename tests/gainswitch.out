0 0 0 0 32 4 0 2 5 0 0 64 0 0 81 255 15 241 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 64 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 255 0 0 0 0 0 0 0 16 0 0 0 0 0 0 0 0 0 0 0 64 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 255 255 255 0 0 0 0 0 0 8 0 0 0 0 0 0 0 0 0 0 0 0 0 45 1 0 16 12 0 0 0 0 0 64 0 0 7 1 0 0 0 0 0 0 0 0 0 0 0 1 19 0 16 129 5 61 1 0 16 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0